packages/common-db/tsconfig.build.json

```diff
@@ -0,0 +1,16 @@
+// Specialized tsconfig to only compile .ts-files in the src dir
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "allowJs": false,
+    "checkJs": false,
+    "noEmit": false,
+    "declaration": false
```

Comment:
`declaration` should (must) be `true`. This makes it possible to use project references in a monorepo.